content/docs/user-guide/packaging/windows-release-builds.md

```diff
@@ -208,65 +216,73 @@ cd C:\o3de-install
 scripts\o3de.bat register --project-path C:\o3de-projects\MyProject
 ```
 
-#### Option 1: Non-monolithic
+Now, you will choose a build type for your project game release layout.
 
-A pre-built SDK engine supports non-monolithic projects by default. As detailed in [Prerequisites](#prerequisites), you should have already created a Visual Studio project for the engine and registered the engine.
+#### Option 1: Monolithic (Game only)
 
-1. Reconfigure your source engine with the `-D` option, `LY_PROJECTS`, pointing to your project's path.
+The most common workflow is to release your game only, in which case you should choose the **monolithic** build. This will build only your game/server launcher and any necessary dependencies.
+
+1. Use CMake to create a Visual Studio project for an engine that supports monolithic projects. Specify a new CMake build directory (`build\windows_mono`) in your pre-built SDK engine directory, separate from your non-monolithic build directory. Specify a monolithic build by enabling the `-D` option, `LY_MONOLITHIC_GAME`.
```

Comment:
Actually  there are some critical missing instructions in this section.

For a Pre-Built SDK is it assumed the user already has the monolithic artifacts as part of the SDK?

According to the instruction it assumes the user does not.
The configuration command and build command below would create a windows monolithic permutation.

That would be added to the SDK layout folder.

From there the user can cmake configure their O3DE project with the updated SDK folder in a project-centric workflow to generate a solution that can create the project game release layout.

I think we can have a blurb as follows:
> Now that the monolithic build artifacts have been added to the SDK engine layout, your project can configure against that layout to build mono.
> Now switch to your project's source directory and use CMake to create a Visual Studio project for the project that builds against the monolithic engine
```
cd C:\o3de-projects\MyProject
cmake --B build\windows_mono -S , -DCMAKE_INSTALL_PREFIX=C:\o3de-projects\MyProjectGameLayout`
```

Afterwards invoke the CMake build wrapper to build the project game release layout using the `install` target.
```
cmake ---build build\windows_mono --target install --config release
```

Reply:
Thanks, will fix!